src/main/java/net/sf/jabref/external/FindFullTextAction.java

```diff
@@ -93,7 +95,7 @@ public void update() {
             basePanel.output(Localization.lang("Finished downloading full text document"));
         }
         else {
-            String message = Localization.lang("Full text article download failed");
+            String message = Localization.lang("Full text document download failed");
```

Comment:
For consistency I think document should be there (and that document is the correct word). Many of the translations already use document by the way.